services/traefik/docker-compose.yml.j2

```diff
@@ -137,6 +143,12 @@ services:
         - traefik.http.middlewares.authenticated_platform_user.forwardauth.address=http://${WEBSERVER_HOST}:${WEBSERVER_PORT}/v0/auth:check
         - traefik.http.middlewares.authenticated_platform_user.forwardauth.trustForwardHeader=true
         - traefik.http.middlewares.authenticated_platform_user.forwardauth.authResponseHeaders=Set-Cookie,osparc-sc2
+        #
+        # middleware: http(s)://(www.) to  https://
```

Comment:
```suggestion
        # middleware: http(s)://(www.) to  http(s)://
```

Reply:
I believe the middleware below does not perform http --> https redirect

Reply:
true, but there is another middleware taking care of this